src/CZ.Azure.FileExchange/Client/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "cz.azure.fileexchange.client",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "snowpack dev --reload",
+    "watch": "snowpack build --watch",
+    "build": "snowpack build"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@snowpack/plugin-sass": "1.4.0",
+    "snowpack": "3.8.8"
+  },
+  "dependencies": {
+    "wavesurfer.js": "5.2.0"
```

Comment:
shouldn't we have only dev dependencies because we don't have a real JS project?

Reply:
this file is gone